usaspending_api/spending_explorer/v2/filters/type_filter.py

```diff
@@ -90,6 +90,9 @@ def type_filter(_type, filters, limit=None):
             'Valid Types: budget_function, budget_subfunction, federal_account, program_activity,'
             'object_class, recipient, award, award_category agency, agency_type, agency_sub')
 
+    if filters is None:
+        raise InvalidParameterException('Missing Required Request Parameter, "fiters": { "filter_options" }')
+
```

Comment:
Thanks for your contribution! Did you check that this raises the exception before the point in the code where the 500 error comes up? Ideally there should be an integration test to prove this functionality in usaspending_api/spending_explorer/tests/integration/test_spending_explorer.py

Reply:
I did not test this out I just assumed that this works because when "types" is missing the exception gets raised and the "filters" property isn't used before that point. However, I can test it this weekend probably when I have time and add a test.